cvs/objects/util/base/source/Makefile

```diff
@@ -14,6 +14,7 @@ OBJS       = atom.o \
              model_time.o \
              summary.o \
              supply_demand_curve.o \
+             supply_demand_curve_saver.o \
```

Comment:
This reminds me, we will need to update the Xcode and Visual Studio project files as well.  I can help with that.